core/src/main/java/org/frankframework/core/Adapter.java

```diff
@@ -743,8 +747,7 @@ public void start() {
 		Runnable runnable = new Runnable() {
 			@Override
 			public void run() {
-				Thread.currentThread().setName("starting Adapter "+getName());
-				try {
+				try (final CloseableThreadContext.Instance ctc = CloseableThreadContext.put(LogUtil.MDC_ADAPTER_KEY, getName())) {
```

Comment:
Looks a lot cleaner in the logs, and helps with structured logging. Cannot use this in the stop method though 😢 .

Reply:
For Erik, this now works in the `stop` method too.